autotest/units/001_one_port/077_bird/autotest.yaml

```diff
@@ -0,0 +1,8 @@
+steps:
+- bird: 
+  - bird/bird.conf
+- birdc:
+  - show route
+- sleep: 20
```

Comment:
is 20 second sleep absolutely needed? I would like to see a lesser value though if it's necessary, then feel free to ignore